### PR TITLE
Add startYear to example build

### PR DESCRIPTION
**Pull request by valencik**
Following https://github.com/typelevel/typelevel.g8/pull/28

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ ThisBuild / tlBaseVersion := "0.4" // your current series x.y
 
 ThisBuild / organization := "org.typelevel"
 ThisBuild / organizationName := "Typelevel"
+ThisBuild / startYear := Some(2022)
```
Comment (armanbilge) [inline docs/index.md:52]: I wanna make this a variable :P

Comment (valencik) [reply docs/index.md:52]: Haha, so that it's always the current year?

Comment (armanbilge) [reply docs/index.md:52]: Right! Anyway feel free to skip, I'll do it while I muck about with coursier 😏

Comment (valencik) [reply docs/index.md:52]: Sounds good, you have 193 days until it becomes an issue 😆

**Pull request merged**